src/pages/UserFeed.js

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+import { useState, useEffect, useContext } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Context } from '../App';
+import VideoFeedItem from '../components/UserFeedItem';
+import UserInfo from '../components/UserInfo';
+import { userApi } from '../api-config/api';
+
+const getUserInfoData = (currentUser) => {
+  console.log(currentUser);
+  return axios
+    .request({
+      method: 'GET',
+      url: `https://tiktok33.p.rapidapi.com/user/info/${currentUser}`,
+      headers: {
+        'x-rapidapi-host': 'tiktok33.p.rapidapi.com',
+        'x-rapidapi-key': userApi
+      }
+    })
+    .then(function (response) {
+      return response.data;
+    })
+    .catch(function (error) {
+      console.error(error);
+    });
+};
+
+const UserFeed = () => {
+  const MAX_POSTS = 29;
```

Comment:
назва цієї змінної суперечить правилу іменування змінних в середені функції

```suggestion
  const maxPosts = 29;
```